egs/tedlium3/asr1/run.sh

```diff
@@ -232,22 +187,27 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
         --train-label ${lmdatadir}/train.txt \
         --valid-label ${lmdatadir}/valid.txt \
         --resume ${lm_resume} \
-        --layer ${lm_layers} \
-        --unit ${lm_units} \
-        --opt ${lm_opt} \
-        --sortagrad ${lm_sortagrad} \
-        --batchsize ${lm_batchsize} \
-        --epoch ${lm_epochs} \
-        --patience ${lm_patience} \
-        --maxlen ${lm_maxlen} \
         --dict ${dict}
 fi
 
+if [ -z ${tag} ]; then
+    expname=${train_set}_${backend}_nbpe${nbpe}_ngpu${ngpu}_$(basename ${train_config%.*})_$(basename ${preprocess_config%.*})
+    if ${do_delta}; then
+        expname=${expname}_delta
+    fi
```

Comment:
Could you add following part? 
https://github.com/espnet/espnet/blob/e8e43af565c4e0fb7b8b6f680c1ede485a5378f7/egs/librispeech/asr1/run.sh#L208-L210